Content.Server/Storage/EntitySystems/StorageSystem.cs

```diff
@@ -601,8 +603,13 @@ public void OpenStorageUI(EntityUid uid, EntityUid entity, ServerStorageComponen
             if (!Resolve(uid, ref storageComp) || !TryComp(entity, out ActorComponent? player))
                 return;
 
+            // prevent spamming bag open / honkerton honk sound
+            silent |= _useDelay.ActiveDelay(uid);
             if (!silent)
+            {
                 _audio.PlayPvs(storageComp.StorageOpenSound, uid);
+                _useDelay.BeginDelay(uid);
```

Comment:
get the use delay comp here and feed it to the methods so it resolves once instead of twice